docs/framework/unmanaged-api/diagnostics/isymunmanagedbinder-interface.md

```diff
@@ -37,7 +37,7 @@ Represents a symbol binder for unmanaged code.
   
 |Method|Description|  
 |------------|-----------------|  
-|[GetReaderForFile Method](../../../../docs/framework/unmanaged-api/diagnostics/isymunmanagedbinder-getreaderforfile-method.md)|Given a metadata interface and a file name, returns the correct <<!--zz xref:ISymUnmanagedReader --> `ISymUnmanagedReader`> structure that will read the debugging symbols associated with the module.|  
+|[GetReaderForFile Method](../../../../docs/framework/unmanaged-api/diagnostics/isymunmanagedbinder-getreaderforfile-method.md)|Given a metadata interface and a file name, returns the correct <xref:ISymUnmanagedReader> structure that will read the debugging symbols associated with the module.|  
```

Comment:
Change <xref:ISymUnmanagedReader> to `[ISymUnmanagedReader](isymunmanagedreader-interface.md)` like in the previous topic